etk/etk.py

```diff
@@ -16,7 +17,7 @@
 class ETK(object):
     def __init__(self, kg_schema=None, modules=None, extract_error_policy="process", logger=None,
                  logger_path=os.path.join(TEMP_DIR, 'etk.log'), ontology=None, generate_json_ld=False,
-                 output_kg_only=False):
+                 output_kg_only=False, use_spacy_tokenizer=False):
```

Comment:
Usually instead of flags I would recommend using an enum pattern, eg prefer passing tokenizer='spacy' over use_spacy_tokenizer=True